index.html

```diff
@@ -1,252 +1,248 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
-      rel="stylesheet"
-    />
+    <!-- <link rel="preconnect" href="https://fonts.googleapis.com" /> -->
+    <!-- <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin /> -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
     <title>Microverse Portofolio project</title>
 </head>
 
 <body>
-    <header>
-        <a class="logo">Dorcas</a>
-        <nav class="Union">
-            <img src="img/Union.png">
-        </nav>
-    </header>
-    <main class="content">
-        <section class="subcontent">
-             <h1>
-                I'm Dorcas
-                 Glad to see 
-                 you!
-             </h1>
-                 
-                 <p>
+    <section class="container">
```

Comment:
[OPTIONAL] Kindly make sure you are following the Figma template for all sizes. The height of this section does not match with the Figma template. Please fix that. 👍 Also background image has a really low resolution it looks pixelated, kindly fic that.
<img width="955" alt="Screenshot 2023-02-24 at 11 02 08 PM" src="https://user-images.githubusercontent.com/25887253/221279753-8315adfa-f408-48d1-b646-a81fbf4dd2b5.png">
 
<img width="1440" alt="Screenshot 2023-02-24 at 11 00 18 PM" src="https://user-images.githubusercontent.com/25887253/221279744-131d7ff8-9126-400f-b160-f3ed4541109e.png">